.scripts/util.sh

```diff
@@ -38,85 +38,88 @@ copy_modules() {
 	local MCHDIR="$1/megast"
 	mkdir -p "$MCHDIR"
-	cp "$SRC/sys/sockets/xif/lance.xif" "$MCHDIR/lance.xix"
-	cp "$SRC/sys/sockets/xif/rieblmst.xif" "$MCHDIR/riebl.xix"
-	cp "$SRC/sys/sockets/xif/rieblmst_fast.xif" "$MCHDIR/riebl_fast.xix"
-	cp "$SRC/sys/sockets/xif/rieblspc.xif" "$MCHDIR/rieblspc.xix"
-	cp "$SRC/sys/sockets/xif/rieblspc_fast.xif" "$MCHDIR/riebls_fast.xix"
-	cp "$SRC/sys/sockets/xif/rtl8012st.xif" "$MCHDIR/rtl8012st.xix"
-	cp "$SRC/sys/xdd/mfp/mfp.xdd" "$MCHDIR"
+	cp "$SRC/sys/sockets/xif/lance_000.xif" "$MCHDIR/lance.xix"
+	cp "$SRC/sys/sockets/xif/rieblmst_000.xif" "$MCHDIR/riebl.xix"
+	cp "$SRC/sys/sockets/xif/rieblmst_fast_000.xif" "$MCHDIR/riebl_fast.xix"
+	cp "$SRC/sys/sockets/xif/rieblspc_000.xif" "$MCHDIR/rieblspc.xix"
+	cp "$SRC/sys/sockets/xif/rieblspc_fast_000.xif" "$MCHDIR/riebls_fast.xix"
+	cp "$SRC/sys/sockets/xif/rtl8012st_000.xif" "$MCHDIR/rtl8012st.xix"
+	cp "$SRC/sys/xdd/mfp/.compile_000/mfp.xdd" "$MCHDIR"
 }
 copy_ste_modules() {
 	local MCHDIR="$1/ste"
 	mkdir -p "$MCHDIR"
-	cp "$SRC/sys/sockets/xif/rtl8012st.xif" "$MCHDIR/rtl8012st.xix"
-	cp "$SRC/sys/xdd/mfp/mfp.xdd" "$MCHDIR"
+	cp "$SRC/sys/sockets/xif/rtl8012st_000.xif" "$MCHDIR/rtl8012st.xix"
+	cp "$SRC/sys/xdd/mfp/.compile_000/mfp.xdd" "$MCHDIR"
 }
 copy_megaste_modules() {
 	local MCHDIR="$1/megaste"
 	mkdir -p "$MCHDIR"
-	cp "$SRC/sys/sockets/xif/rieblste.xif" "$MCHDIR/rieblste.xix"
-	cp "$SRC/sys/sockets/xif/rtl8012st.xif" "$MCHDIR/rtl8012st.xix"
-	cp "$SRC/sys/xdd/mfp/mfp.xdd" "$MCHDIR"
-	cp "$SRC/sys/xdd/scc/scc.xdd" "$MCHDIR"
+	cp "$SRC/sys/sockets/xif/rieblste_000.xif" "$MCHDIR/rieblste.xix"
+	cp "$SRC/sys/sockets/xif/rtl8012st_000.xif" "$MCHDIR/rtl8012st.xix"
+	cp "$SRC/sys/xdd/mfp/.compile_000/mfp.xdd" "$MCHDIR"
+	cp "$SRC/sys/xdd/scc/.compile_000/scc.xdd" "$MCHDIR"
 }
 copy_tt_modules() {
 	local MCHDIR="$1/tt"
+	local TARGET="$2"
 	mkdir -p "$MCHDIR"
-	cp "$SRC/sys/sockets/xif/daynaport/scsilink.xif" "$MCHDIR/scsilink.xix"
-	cp "$SRC/sys/sockets/xif/lance.xif" "$MCHDIR/lance.xix"
-	cp "$SRC/sys/sockets/xif/riebltt.xif" "$MCHDIR/riebltt.xix"
-	cp "$SRC/sys/sockets/xif/rtl8012.xif" "$MCHDIR/rtl8012.xix"
-	cp "$SRC/sys/xdd/mfp/mfp.xdd" "$MCHDIR"
-	cp "$SRC/sys/xdd/scc/scc.xdd" "$MCHDIR"
+	cp "$SRC/sys/sockets/xif/daynaport/.compile_030/scsilink.xif" "$MCHDIR/scsilink.xix"
+	cp "$SRC/sys/sockets/xif/lance_030.xif" "$MCHDIR/lance.xix"
+	cp "$SRC/sys/sockets/xif/riebltt_030.xif" "$MCHDIR/riebltt.xix"
+	cp "$SRC/sys/sockets/xif/rtl8012_030.xif" "$MCHDIR/rtl8012.xix"
+	cp "$SRC/sys/xdd/mfp/.compile_030/mfp.xdd" "$MCHDIR"
+	cp "$SRC/sys/xdd/scc/.compile_030/scc.xdd" "$MCHDIR"
 }
 copy_falcon_modules() {
 	local MCHDIR="$1/falcon"
 	mkdir -p "$MCHDIR"
-	cp "$SRC/sys/sockets/xif/rtl8012.xif" "$MCHDIR/rtl8012.xix"
-	cp "$SRC/sys/sockets/xif/daynaport/scsilink.xif" "$MCHDIR/scsilink.xix"
-	cp "$SRC/sys/xdd/dsp56k/dsp56k.xdd" "$MCHDIR"
-	cp "$SRC/sys/xdd/scc/scc.xdd" "$MCHDIR"
+	cp "$SRC/sys/sockets/xif/rtl8012_02060.xif" "$MCHDIR/rtl8012.xix"
```

Comment:
While this is not wrong, it could be done in a better way, as discussed in the other thread:

- for `freemint-latest-02060-tt_falcon_clones.zip` 02060 is great (as it can be booted on both 030 and 040 Falcons; the AB040 has no separate `MCHDIR` ... which maybe should)
- for `freemint-latest-030.zip` and `freemint-latest-040.zip` there's no reason why the drivers shouldn't be compiled directly for given CPU